gnome-initial-setup/pages/summary/gis-summary-page.ui

```diff
@@ -27,6 +27,42 @@
                 <property name="height_request">96</property>
               </object>
             </child>
+            <child>
+              <object class="GtkBox" id="hack_box">
+                <property name="visible">False</property>
+                <property name="halign">center</property>
+                <property name="valign">center</property>
+                <property name="margin_left">20</property>
+                <property name="margin_right">20</property>
+                <property name="orientation">horizontal</property>
+                <property name="spacing">12</property>
+                <child>
+                  <object class="GtkImage">
+                    <property name="visible" bind-source="image" bind-property="visible" bind-flags="invert-boolean|sync-create|bidirectional">False</property>
```

Comment:
Interesting approach – does this work as expected given it's the visibility of `hack_box` that you're adjusting in the code, not its children?